src/main/java/com/datalogics/pdf/hsm/samples/LunaHsmManager.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2016 Datalogics Inc.
+ */
+
+package com.datalogics.pdf.hsm.samples;
+
+import com.adobe.pdfjt.core.credentials.CredentialFactory;
+import com.adobe.pdfjt.core.credentials.Credentials;
+import com.adobe.pdfjt.core.credentials.PrivateKeyHolder;
+import com.adobe.pdfjt.core.credentials.PrivateKeyHolderFactory;
+import com.adobe.pdfjt.core.exceptions.PDFInvalidParameterException;
+
+import com.safenetinc.luna.LunaSlotManager;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Enumeration;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+/**
+ * This class allows for connecting to a Luna SA Hsm Device.
+ */
+public final class LunaHsmManager implements HsmManager {
+    private static final Logger LOGGER = Logger.getLogger(LunaHsmManager.class.getName());
+
+    private LunaSlotManager slotManager;
+
+
+    /**
+     * Default no-arg constructor.
+     */
+    protected LunaHsmManager() {
+        super();
+    }
+
+    /*
+     * (non-Javadoc)
+     *
+     * @see com.datalogics.pdf.hsm.samples.HsmManager#hsmLogin(java.lang.String, java.lang.String)
+     */
+    @Override
+    public boolean hsmLogin(final String tokenLabel, final String password) {
+        // Initialize the SlotManager class
+        slotManager = LunaSlotManager.getInstance();
+
+        // Login to the HSM
+        /*
+         * This method unlocks the token for use. There are multiple methods available within the LunaSlotManager class
+         * to login to the HSM:
+         *
+         * Login to the first available partition: login(String password)
+         *
+         * Login to the partition at the specified slot: login(int slot, String password)
+         *
+         * Login to the partition with the specified label: login(String TokenLabel, String password)
+         *
+         * Login to the partition with the specified user role: login(int slot, UserRole userType, String password)
+         *
+         * The password argument is the challenge password for the Luna SA partition or HSM. (Applications generally ask
+         * for password information interactively from the user.)
+         */
```

Comment:
Correct this would be more for our benefit than the customer because this is just an example of connecting to our (Luna) HSM which the customer may or may not being using the same brand. 

And if they're using a Luna HSM they probably have access to the Javadocs for Luna and can look up how to use the LunaSlotManager.

I'll remove these comments. Thanks!